jcsv.go

```diff
@@ -1,11 +1,80 @@
 package jcsv
 
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+)
+
 func JsonToCsv(j []byte, addHeaders bool) ([]byte, error) {
-	// TODO: convert JSON data in `j` into CSV format and return
-	return nil, nil
+	// convert []byte to file struct
+	file, err := ParseJson(j)
+	if err != nil {
+		return nil, err
+	}
+
+	// return converted CSV in []byte form
+	return file.Csv(addHeaders), nil
 }
 
 func CsvToJson(c []byte, hasHeaders bool) ([]byte, error) {
-	// TODO: convert CSV data in `c` into JSON format and return
-	return nil, nil
+
+	// split the string on the basis of '\n'
```

Comment:
Why don't you do it like how you did `JsonToCsv()`?

Reply:
It was working fine so I did not change that code. 
I will change it. Is there any built-in method in go that supports conversion of []byte to [][]string and [][]string to []byte?